src/components/FormStart/tests.spec.js

```diff
@@ -75,3 +88,24 @@ it('Form start does not start if there are auth errors', () => {
     expect(onFormStart).toHaveBeenCalledTimes(0);
   }
 });
+
+it('Form start page does not show login buttons if an active submission is present', () => {
+  useQuery.mockReturnValue(new URLSearchParams());
+  const onFormStart = jest.fn();
+  const onFormAbort = jest.fn();
+
+  renderTest(
+    <IntlProvider locale="nl" messages={messagesNL}>
+      <FormStart
+        form={testForm}
+        onFormStart={onFormStart}
+        onFormAbort={onFormAbort}
+        hasActiveSubmission={true}
+      />
+    </IntlProvider>,
+    container
+  );
+
+  expect(screen.queryByText('Continue existing submission')).toBeInTheDocument();
```

Comment:
```suggestion
  expect(screen.queryByRole('button', {name: 'Continue existing submission'})).toBeInTheDocument();
```

See https://testing-library.com/docs/queries/about#priority - if you can't query by accessible role, this is often an indication that there is an a11y issue.